pkg/services/sqlstore/sqlstore.go

```diff
@@ -177,6 +198,18 @@ func LoadConfig() {
 		DbCfg.Host = sec.Key("host").String()
 		DbCfg.Name = sec.Key("name").String()
 		DbCfg.User = sec.Key("user").String()
+		DbCfg.MaxConn, err = sec.Key("max_conn").Int()
```

Comment:
Please use `.MustInt(0)` instead. Reduce the amount of code needed to read the setting. 0 is the default value anyway :)